features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/monitor/DatadogRumMonitor.kt

```diff
@@ -98,6 +99,17 @@ internal class DatadogRumMonitor(
 
     // region RumMonitor
 
+    override val currentSessionId: String?
```

Comment:
This method is not thread-safe, because all the values inside RUM scopes hierarchy are set from the dedicated RUM thread, but `currentSessionId` can be called from any user thread. IMO it is not a big deal here, but if use-case from the user side requires consistency and precision, we may need to make this one thread-safe to avoid concurrency issues.